api-reference/beta/api/profilephoto_get.md

```diff
@@ -4,28 +4,27 @@
 
 Get the specified [profilePhoto](../resources/profilephoto.md) or its metadata (**profilePhoto** properties).
 
-A GET photo operation first looks for the specified photo in the user's mailbox on Exchange Online, and if 
-it's not available there, then looks in Azure Active Directory (AAD).
+A GET photo operation first attempt to retrieve the specified photo from Office 365. If the photo is not available in Office 365, the API attempts to retrieve the photo from Azure Active Directory.
 
-The supported sizes of HD photos on Exchange Online are as follows: '48x48', '64x64', '96x96', '120x120', '240x240', 
-'360x360','432x432', '504x504', and '648x648'. Photos can be any dimension if they are stored in AAD.
+The supported sizes of HD photos on Office 365 are as follows: '48x48', '64x64', '96x96', '120x120', '240x240', 
+'360x360','432x432', '504x504', and '648x648'. Photos can be any dimension if they are stored in Azure Active Directory.
 
 You can get the metadata of the largest available photo, or specify a size to get the metadata for that photo size.
 If the size you request is not available, you can still get a smaller size that the user has uploaded and made available.
 For example, if the user uploads a photo that is 504x504 pixels, then all but the 648x648 size of photo will be available for download.
-If the specified size is not available in the user's mailbox or in AAD, the size of '1x1' is returned with the rest of 
+If the specified size is not available in the user's mailbox or in Azure Active Directory, the size of '1x1' is returned with the rest of 
 metadata.
 
-> **Note** The GET photo operation in beta supports a user's work, school, or personal mailboxes. The GET photo metadata operation,
-however, supports only the user's work or school mailboxes and not personal mailboxes.
```

Comment:
@dkershaw10 as far as I remember photos for users with their personal mailboxes are not yet on v1.0 and are only on beta endpoint. Therefore, this note should not be deleted from this page for now.